docs/uml-class-diagrams/display01/test/DisplayGpsManagerImplementationTests/DisplayGpsManagerImplementationTests.puml

```diff
@@ -2,8 +2,11 @@
 
 package edu.oakland.test.display01 {
     class DisplayGpsManagerImplementationTests {
-
+        + void receivedGpsSignalStrengths()
+        + void ifAcceptableGpsSignalStrengths()
+        + void elsePoorGpsSignalStrengths()
```

Comment:
As we start building we may not need these two tests, but let's leave them in for now just in case.